ajax/libs/jquery.loadTemplate/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "jquery.loadTemplate",
+    "description": "A template plugin for jQuery. Allows templating without cluttering JavaScript code with markup",
+    "filename": "jquery.loadTemplate.min.js",
+    "keywords": [
+       "templates",
+       "templating",
+       "jquery-plugin"
+    ],
+    "author": "codepb",
+    "homepage": "http://codepb.github.io/jquery-template/",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com:codepb/jquery-template.git"
+    },
+    "npmName": "jquery.loadtemplate ",
+    "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/jquery.loadTemplate"        
```

Comment:
`files` is a pattern matcher allowing selection of multiple files, you could refer to [node-glob](https://github.com/isaacs/node-glob#glob-primer) to know more that what patterns can use.
So this pattern can't get any files.

Because the basePath is `dist`, which is the files of the library publish to web front-end developer use, let's use `**/*`.
Thank you.